EpiAware/docs/src/getting-started/faq.md

```diff
@@ -10,8 +10,11 @@ Pages = ["lib/getting-started/faq.md"]
 
 We use [`Pluto.jl`](https://plutojl.org/) scripts as part of our documentation and testing. The scripts are located in `docs/src/examples` and can be run using the `Pluto.jl` package.
 We recommend using the version of `Pluto` that is pinned in the `Project.toml` file defining the documentation environment.
+
```

Comment:
i.e this line below seems like enough if we mention it in the developer docs vs here? 

or mention it here and link out from the developer docs?